labellab-client/src/components/login/ResetPassword.js

```diff
@@ -18,7 +18,7 @@ class ResetPassword extends Component {
       updated:false
     };
   }
-
+F
```

Comment:
there is a additional F letter i think :slightly_smiling_face:

Reply:
done!